### PR TITLE
Add add-on installation to voice setup flow

### PR DESCRIPTION
**Pull request by bramkragten**
## Proposed change

Adds the installation and setup of Whisper and Piper add-ons to the voice wizard flow.

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [X] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/dialogs/voice-assistant-setup/voice-assistant-setup-step-local.ts [hunk 1]
```diff
@@ -0,0 +1,345 @@
...
+
+              <a
+                href=${documentationUrl(
+                  this.hass,
+                  "/voice_control/voice_remote_local_assistant/"
+                )}
+                target="_blank"
+                rel="noreferrer noopenner"
+              >
+                <ha-button @click=${this._skip}>
```
Comment (wendevlin) [inline src/dialogs/voice-assistant-setup/voice-assistant-setup-step-local.ts:72]:
```suggestion
                <ha-button>
```

Why would I skip when I want to learn more?

Comment (bramkragten) [reply src/dialogs/voice-assistant-setup/voice-assistant-setup-step-local.ts:72]: Because there is no other way to skip, and nothing else to do

Comment (wendevlin) [reply src/dialogs/voice-assistant-setup/voice-assistant-setup-step-local.ts:72]: There is the skip button on the top right...

Comment (wendevlin) [reply src/dialogs/voice-assistant-setup/voice-assistant-setup-step-local.ts:72]: Sorry I looked on the wrong view

File: src/dialogs/voice-assistant-setup/voice-assistant-setup-step-pipeline.ts [hunk 2]
```diff
@@ -117,19 +117,24 @@ export class HaVoiceAssistantSetupStepPipeline extends LitElement {
...
+          <a
+            href=${documentationUrl(
+              this.hass,
+              "/voice_control/voice_remote_local_assistant/"
+            )}
+            target="_blank"
+            rel="noreferrer noopenner"
           >
-        </a>
+            <ha-button @click=${this._skip}>
```
Comment (wendevlin) [inline src/dialogs/voice-assistant-setup/voice-assistant-setup-step-pipeline.ts:129]:
```suggestion
            <ha-button>
```

I meant this button, we have a skip in top right.

File: src/dialogs/voice-assistant-setup/voice-assistant-setup-step-local.ts [hunk 3]
```diff
@@ -0,0 +1,345 @@
...
+                for you. Read the documentation to learn how to install them.
+              </p>
+
+              <a
+                href=${documentationUrl(
+                  this.hass,
+                  "/voice_control/voice_remote_local_assistant/"
+                )}
+                target="_blank"
+                rel="noreferrer noopenner"
```
Comment (wendevlin) [inline src/dialogs/voice-assistant-setup/voice-assistant-setup-step-local.ts:70]:
```suggestion
                rel="noreferrer noopener"
```

File: src/dialogs/voice-assistant-setup/voice-assistant-setup-step-local.ts [hunk 4]
```diff
@@ -0,0 +1,345 @@
...
+                  TTS and STT provider. Learn how to set up local TTS and STT
+                  providers in the documentation.
+                </p>
+                <a
+                  href=${documentationUrl(
+                    this.hass,
+                    "/voice_control/voice_remote_local_assistant/"
+                  )}
+                  target="_blank"
+                  rel="noreferrer noopenner"
```
Comment (wendevlin) [inline src/dialogs/voice-assistant-setup/voice-assistant-setup-step-local.ts:91]:
```suggestion
                  rel="noreferrer noopener"
```

I think there are 2 more in the code, maybe fix all.

File: src/dialogs/voice-assistant-setup/voice-assistant-setup-step-local.ts [hunk 5]
```diff
@@ -0,0 +1,345 @@
...
+                >
+                  <ha-button @click=${this._nextStep}>
+                    <ha-svg-icon
+                      .path=${mdiOpenInNew}
+                      slot="icon"
+                    ></ha-svg-icon>
+                    Learn more</ha-button
+                  > </a
+                >.`
```
Comment (wendevlin) [inline src/dialogs/voice-assistant-setup/voice-assistant-setup-step-local.ts:100]:
```suggestion
                >`
```

File: src/dialogs/voice-assistant-setup/voice-assistant-setup-step-local.ts (hunk 1 above)
Comment (bramkragten) [reply src/dialogs/voice-assistant-setup/voice-assistant-setup-step-local.ts]: I changed it now, added back button when failed.

**Review by wendevlin — APPROVED** [review]

**Pull request merged**